bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/checksum/DummyDigestManager.java

```diff
@@ -0,0 +1,50 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.apache.bookkeeper.proto.checksum;
+
+import io.netty.buffer.ByteBuf;
+
+
+/**
+ * This class provides a noop digest implementation.
+ */
+public class DummyDigestManager extends DigestManager {
+    private static final byte[] dummyDigest = new byte[] { 0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x6, 0x07 };
+
+    public DummyDigestManager(long ledgerId) {
+        super(ledgerId);
+    }
+
+    @Override
+    int getMacCodeLength() {
+        return 8;
```

Comment:
any reason why a dummy digest manager needs a 8 bytes here? can't we just make it 0? the digest here is anyway useless, no?

Reply:
Not sure.  Perhaps there is value in emulating the size overhead?  @jvrao Thoughts?

Reply:
I really don't really recall why we made up the digest and kept it at 8. Maybe tried to keep the payload size in sync with CRC32.

Reply:
@jvrao any idea to waste extra 4 bytes, if it is not even used.

Reply:
@jvrao I'll switch it to an empty payload.

Reply:
I am fine with it.